### PR TITLE
✌️ يونس التركي

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: IAdapter.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+using static System.Console;
+namespace adapter_dp
+{
+    public class IAdapter
```
Comment (nm-safcsp) [inline IAdapter.cs:4]: `IAdapter` should be an interface

File: IAdapter.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    {
+        public float Boiling { get; set; }
+        public float Melting { get; set; }
+        public double Weight { get; set; }
+        public string Formula { get; set; }
+        public virtual void Print()
+        {
+            WriteLine("\nCompound: Unknown ------ ");
```
Comment (nm-safcsp) [inline IAdapter.cs:12]: Nice

**Review by nm-safcsp — COMMENTED** [review]
**Amazing job Younes!! :clap::clap:**
You have used a great idea to showcase how the Adapter design pattern solves problems! Thank you for your creative work.
But I noticed that your code is very similar to "Software Design Patterns C#.NET by K.P.P. Suneth Pathirana | 6. Adapter Design Pattern".
Thank you.

> don't forget to check the comments on your code.

**Comment by nm-safcsp** [comment]
**Amazing job Younes!! :clap::clap:**
You have used a great idea to showcase how the Adapter design pattern solves problems! Thank you for your creative work.
But I noticed that your code is very similar to "Software Design Patterns C#.NET by K.P.P. Suneth Pathirana | 6. Adapter Design Pattern".
Thank you.

> don't forget to check the comments on your code.

File: IAdapter.cs (hunk 1 above)
Comment (ghost) [reply IAdapter.cs:4]: > **Amazing job Younes!! 👏👏**
> You have used a great idea to showcase how the Adapter design pattern solves problems! Thank you for your creative work.
> But I noticed that your code is very similar to "Software Design Patterns C#.NET by K.P.P. Suneth Pathirana | 6. Adapter Design Pattern".
> Thank you.
> 
> > don't forget to check the comments on your code.

Thank you Norah.

Yes, my solution is based on that article. I was looking to practice not find creative use for the dp like other homework.

Comment (nm-safcsp) [reply IAdapter.cs:4]: Good approach! but keep in mind that in future you have to reference any resource you have used.

Comment (ghost) [reply IAdapter.cs:4]: Yes, Ma'am!

**Pull request closed without merging**